src/app/login/login.module.ts

```diff
@@ -23,16 +23,17 @@ import {LoginFormComponent} from './components/login-form/login-form.component';
   ],
   imports: [
     CommonModule,
-    BrowserAnimationsModule,
     ReactiveFormsModule,
     MatFormFieldModule,
     MatButtonModule,
     MatInputModule,
     MatIconModule,
-    TranslateModule
+    TranslateModule,
+    RouterModule.forChild([{ path: '', component: LoginComponent }])
```

Comment:
this line of code is correct but I have never seen routes defined in this way. It should be in a separate "routes" variable IMO.